_posts/2020-01-14-role-assumption-tricks.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: post
+title: AWS STS Role Chaining
+date: 2020-01-14
+tags: javascript tutorial serverless sam
+author: Matthew Tyler
+image: 
+---
+
+<center><img src="/img/serverless-express.png" /></center>
+<br/>
+
+# Introduction
+
+At some in your career using AWS, you'll find it nessecary to learn a little about how assuming roles in other accounts works. If your working on a personal account, chances are you used to logging in with a user that you have created for yourself. When you join a company using AWS, it's more likely that they have a multiple account set up using AWS Organizations with AWS SSO - in which case you will log into a specific account using a role via federated access. Likewise, you are probably used to needing to create roles for various services (like Lambda), and provide a service trust so that the service can use a role.
```

Comment:
Opener is broke

`At some in your career using AWS,`

Reply:
`logging in with a user that you have`
Do you mean, logging in with a `user account`? Or better, an `IAM user account`

Reply:
hey have a multiple account set up (missing S on accounts)